### PR TITLE
Plural fix for accelerations

### PR DESCRIPTION
**Pull request by tmilnthorp**

### CODE REVIEW

File: UnitsNet/UnitDefinitions/Acceleration.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ﻿{
   "Name": "Acceleration",
-  "BaseUnit": "MeterPerSecondSquared",
+  "BaseUnit": "MetersPerSecondSquared",
   "XmlDoc": "Acceleration, in physics, is the rate at which the velocity of an object changes over time. An object's acceleration is the net result of any and all forces acting on the object, as described by Newton's Second Law. The SI unit for acceleration is the Meter per second squared (m/s²). Accelerations are vector quantities (they have magnitude and direction) and add according to the parallelogram law. As a vector, the calculated net force is equal to the product of the object's mass (a scalar quantity) and the acceleration.",
   "Units": [
     {
-      "SingularName": "MeterPerSecondSquared",
-      "PluralName": "MeterPerSecondSquared",
+      "SingularName": "MetersPerSecondSquared",
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/Acceleration.json]: Singular name should stay `MeterPerSecondSquared`.

File: UnitsNet.Tests/CustomCode/AccelerationTests.cs [hunk 2]
```diff
@@ -23,30 +23,30 @@ namespace UnitsNet.Tests.CustomCode
...
+        protected override double StandardGravityInOneMetersPerSecondSquared => 0.1019727;
 
-        protected override double InchesPerSecondSquaredInOneMeterPerSecondSquared => 39.3700787;
+        protected override double InchesPerSecondSquaredInOneMetersPerSecondSquared => 39.3700787;
 
-        protected override double FeetPerSecondSquaredInOneMeterPerSecondSquared => 3.28084;
+        protected override double FeetPerSecondSquaredInOneMetersPerSecondSquared => 3.28084;
 
-        protected override double KnotsPerHourInOneMeterPerSecondSquared => 6.99784017278618E3;
+        protected override double KnotsPerHourInOneMetersPerSecondSquared => 6.99784017278618E3;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/AccelerationTests.cs]: See below, singular name is used here so due to wrong singular name in JSON below it reads `InOneMeters` here.

**Comment by tmilnthorp** [comment]
A bit fat-fingered with find replace :) fixed

**Comment by angularsen** [comment]
Looks good ! :-)

**Pull request merged**